src/util/records.py

```diff
@@ -351,10 +397,13 @@ def remove_record(cr, name):
     if model == "res.groups":
         _logger.log(NEARLYWARN, "Removing group %r", name)
         return remove_group(cr, group_id=res_id)
+    
+    if model in get_models_inheriting_actions_actions(cr):
```

Comment:
We could have a hard-coded list of standard action models. We don't need to take into account all possible models extending the actions base model. We don't that for views for example.

Reply:
We can just use `for_each_inherit`

Reply:
The SQL query version plus the query itself adds complexity, it's cleaner with `for_each_inherit`, IMO.